### PR TITLE
Assessment notebook for December workshop

### PR DESCRIPTION
**Pull request by gmingas**

### CODE REVIEW

**Review by andrewphilipsmith — COMMENTED** [review]
I have reviewed the non-R code so far (eg python and config files).

There are a couple of things that I think need to be addressed. I've added in-line comments for these.

I'll add more comments as I get to them.

File: python/debiasing/debias_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,720 @@
...
+from utils import (
+    DATE_FORMAT_SPLIT_STR,
+    DATE_FORMAT_STR,
+    DateType,
+    date_range_to_str,
+    iter_to_tuple_strs,
+    path_iterdir,
+)
+
+if sys.version_info >= (3, 11):
```
[the remark this answers is not in the document]
Comment (griff-rees) [reply python/debiasing/debias_wrapper.py:19]: Thanks yeah this is a case of future proofing.

File: python/utils.py [hunk 2]
```diff
@@ -105,3 +108,44 @@ def path_iterdir(
             raise error
         else:
             return
+
+
+def make_user(
```
[the remark this answers is not in the document]
Comment (griff-rees) [reply python/utils.py:113]: Documentation of use and intention (running workshops) added in ced234c

File: python/debiasing/debias_wrapper.py [hunk 3]
```diff
@@ -0,0 +1,720 @@
...
+                variable=variable,
+                method=method,
+                input_data_path=input_data_path,
+                cmethods_out_path=cmethods_out_path,
+                processors=processors,
+            )
+        )
+
+
+# @pytest.mark.server
```
[the remark this answers is not in the document]
Comment (griff-rees) [reply python/debiasing/debias_wrapper.py:611]: Should be addressed in ef05e11

**Comment by griff-rees** [comment]
All comments addressed in #122

**Pull request closed without merging**